yjit/src/virtualmem.rs

```diff
@@ -29,6 +29,9 @@ pub struct VirtualMemory<A: Allocator> {
     /// Size of the region in bytes.
```

Comment:
Not directly related to this PR but what do we mean by "region" here? Could we clarify?

Reply:
/region/virtual memory region/s clear enough?

Reply:
applied https://github.com/ruby/ruby/pull/11810#discussion_r1790504977